dev-embedded/u-boot-tools/u-boot-tools-2020.10.ebuild

```diff
@@ -12,59 +12,71 @@ SRC_URI="ftp://ftp.denx.de/pub/u-boot/${MY_P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
```

Comment:
(Commenting on a different line because the review thing doesn't let me comment on a line you didn't change)

Why does the SRC_URI use the ftp protocol? It seems like the distfiles are also served over HTTPS, can you switch to that for security?